.github/workflows/test-future.yml

```diff
@@ -17,22 +17,14 @@ jobs:
         with:
           ref: 'master'
           fetch-depth: 0
-      - uses: actions/setup-python@v5
```

Comment:
Letting `uv` install Python?

[uv documentation](https://docs.astral.sh/uv/guides/integration/github/#setting-up-python) says:
>Alternatively, the official GitHub `setup-python` action can be used. This can be faster, because GitHub caches the Python versions alongside the runner.